cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,8 @@ class ProductListPage {
     private checkoutBtn: string
 
     constructor() {
-        this.tshirtBuy = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-        this.checkoutBtn = "[style*='display: block;'] .button-container > a";
+        this.tshirtBuy = ".button-container [title='Add to cart']";
```

Comment:
probably just `a[title='Add to cart']` should work

Reply:
Thanks :)